<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>UI_page1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QComboBox" name="Tab1ComboParameter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="Tab1ComboTestNames">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Tab1Build">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Построить распределение</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Тест</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Параметр</string>
       </property>
      </widget>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>221</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Tab1TextEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>471</width>
        <height>451</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ограничить даты</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Список плат</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбрать все</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>UI_page2</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>241</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="Tab2ComboBID">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Показать результаты измерения</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Board_ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>№ измерения</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="Tab1TextEdit_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>471</width>
        <height>451</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>UI_page3</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QComboBox" name="comboBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="Tab3ComboBID">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Показать результаты измерения</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Board_ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>№ измерения</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>221</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>471</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>Graph</string>
      </property>
      <widget class="QListWidget" name="listWidget_3">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>171</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>150</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Построить график</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>560</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Задать границы параметров</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
